README.md

```diff
@@ -56,7 +56,7 @@ This **open-source plugin was developed to help the Sylius community**. If you h
 We work on stable, supported and up-to-date versions of packages. We recommend you to do the same.
 If you use Sylius 1.4, you might get a compatibility issue for Pagerfanta. Please read [this issue](https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/issues/23) in order to proceed with a workaround.
 
-*Note*: This Plugin currently supports ElasticSearch 5.3.x up to 6.8.x.  ElasticSearch ^7.x is not currently supported.
+*Note*: This Plugin currently supports ElasticSearch 5.3.x up to 6.8.x.  ElasticSearch ^7.x is supported from version ^2.0.0.
```

Comment:
```suggestion
*Note*: This Plugin supports ElasticSearch 7.0 and above. If you're looking for ElasticSearch Plugin for older versions check SyliusElasticSearchPlugin in version `1.x`.
```

Hi 🙋🏻‍♂️.
Thanks for your PR. As `master` branch is currently for `2.0` version and above I'd prefer to use this sentence. Your version is also great but it fits more in context of `1.0` version. Would you like to open next PR against `1.x` branch :)?

Once again - thank you :)!